common/genes/05_genes_special_accessories_clothes.txt

```diff
@@ -1362,7 +1362,7 @@ special_genes = {
 				girl = female
```

Comment:
Still persist.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/125706220-0cae3cb4-ec11-4f13-9793-4d6523ac3250.png)

</details>

Reply:
Nope, it still has problems. Check how it looks in the Portrait Editor before pushing it.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/127568726-0c067357-39d7-4de2-9a4b-7db083c5a92a.png)

</details>